config/settings/base.py

```diff
@@ -287,13 +287,6 @@
 SOCIALACCOUNT_ALLOW_REGISTRATION = env.bool("DJANGO_SOCIALACCOUNT_ALLOW_REGISTRATION", True)
 ACCOUNT_ALLOWED_EMAIL_DOMAINS = env.list("DJANGO_ACCOUNT_ALLOWED_EMAIL_DOMAINS", default=["opencraft.com"])
 
-# Social
```

Comment:
Why are these settings being removed? Are we passing variables in a different way for the SSO?

Reply:
Yep, there were some problems with the library handling social auth and we switched to a different one. I forgot to remove these settings in #3. Sorry for the noise in this PR!